libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -1002,12 +1002,23 @@ - (void)showAlertWithTitle:(nullable NSString *)title message:(nullable NSString
         
         [self.statusAlert addAction:cancelAction];
         dispatch_async(dispatch_get_main_queue(), ^{
-            [[SFRootViewManager sharedManager] pushViewController:weakSelf.statusAlert];
+            
+            SFSDKWindowManager.sharedManager.authWindow.viewController = [self blankViewController];
+            [SFSDKWindowManager.sharedManager.authWindow enable:YES withCompletion:^{
+                [SFSDKWindowManager.sharedManager.authWindow.viewController presentViewController:weakSelf.statusAlert animated:NO completion:^{
```

Comment:
It looks like this completion block is empty. Can we replace it with `nil` instead?